src/store/metadata/stub/content/datasets.json

```diff
@@ -46,8 +45,49 @@
                 "modified": "2017-02-21T09:30:00Z"
               }
             ]
+        },
+        {
+            "@id": "https://staging.idpd.uk/datasets/4gc",
+            "@type": "dcat:DatasetSeries",
+            "identifier": "4gc",
+            "title": "4G Coverage",
+            "summary": "Percentage of geographic areas with 4G signal outdoors from at least 1 operator (signal threshold: 105dBm), UK, September 2022.",
+            "description": "This dataset shows the percentage of geographic areas with 4G signal outdoors from at least 1 operator (signal threshold: 105dBm), in the UK, for September 2022.\n\n### Coverage \n\nUK\n\n#### Geography definition\n\nLocal Authority Districts and Unitary Authorities\n\n### Source\n\n - United Kingdom: [Ofcom](https://www.ofcom.org.uk/research-and-data/multi-sector-research/infrastructure-research/connected-nations-2022) \n\n\n",
+            "issued": "2023-03-31T12:10:00",
+            "modified": "2023-10-23T15:55:00",
+            "next_release": "2023-11-24T09:30:00Z",
+            "publisher": "https://staging.idpd.uk/publishers/office-for-national-statistics",
+            "creator": "https://staging.idpd.uk/publishers/ofcom",
+            "contact_point": {
+              "name": "Ofcom Open Data",
+              "email": "mailto:open.data@ofcom.org.uk"
+            },
+            "topics": [
+              "https://staging.idpd.uk/topics/communication",
+              "https://staging.idpd.uk/topics/itandinternetindustry"],
+            "frequency": "monthly",
+            "keywords": [
+              "broadband-mobile-coverage",
+              "connectivity",
+              "local-authority",
+              "subnational"
+            ],
+            "licence": "http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/",
+            "spatial_coverage": "K02000001",
+            "temporal_coverage": {
+              "start": "2022-09-01T00:00:00Z",
+              "end": "2022-09-01T00:00:00Z"
+            },
+            "editions_url": "https://staging.idpd.uk/datasets/4gc/editions",
```

Comment:
Can someone please explain what this is? I don't think it would ever be valid json-ld.

Reply:
initial attempt at highlighting navigation to other api resources from that meeting when you were on leave.

i think @eldeal 's work on the _embed thimgy would supercede this, if so its on the way  out anyway.